doc/findinfiles.rst

```diff
@@ -38,21 +39,25 @@ You can select from a number of options to enable searches as broad or refined a
 
 To enable case sensitivity, which only returns matches with identical capitalization to your search text, toggle the :guilabel:`Aa` button on.
 
-[GIF showing search with case sensitive activated]
+.. image:: images/find_in_files/find-in-files-case.gif
+   :alt: Spyder Find pane showing search with case sensitive activated
 
 To parse your search string as a `regular expression`_, enable the :guilabel:`.*` button.
 
 .. _regular expression: https://docs.python.org/3/library/re.html
 
-[GIF showing regular expression search]
+.. image:: images/find_in_files/find-in-files-regex.gif
```

Comment:
Again, around 5 seconds iis spent here with the search chugging, while only for a second at the end is it actually shown that both `editor = DataFrameEditor()` and `model = DataFrameModel()` were matched (the point of the regex). I recommend speeding up the search by 2x, and using ~1s of the 2+s freed to pause longer at the end to show this.

Reply:
Will do

Reply:
Agreed.